bitnami/argo-cd/templates/server/deployment.yaml

```
@@ -169,6 +169,10 @@ spec:
               containerPort: {{ .Values.server.containerPorts.metrics }}
               protocol: TCP
           env:
+            {{- if gt .Values.server.replicaCount 1 }}
```

Numbers coming from values files are considered floats.
```suggestion
            {{- if gt ( int .Values.server.replicaCount ) 1 }}
```